react/CustomWrapper.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Helmet, useRuntime } from 'vtex.render-runtime'
+
+import DataLayerApolloWrapper from './components/DataLayerApolloWrapper'
+
+const CustomWrapper = ({
+  children,
+  metaTitle,
+  metaDescription,
+  metaKeywords,
+}) => {
+  const { account } = useRuntime()
+
+  const getData = () => ({
+    event: 'pageInfo',
+    eventType: 'customView',
+    accountName: account,
+    pageTitle: document.title,
```

Comment:
Any chance that this is executed in SSR?

Reply:
All wrappers have this method. I think it works. 😬